src/Model/users.js

```diff
@@ -0,0 +1,7 @@
+const users = {
```

Comment:
İçinde birden fazla `user` tutmadığımız için isim tekil olmalıdır.
```suggestion
const user = {
```